tests/js/spec/views/performance/transactionDetails/quickTraceMeta.spec.jsx

```diff
@@ -51,7 +60,7 @@ describe('QuickTraceMeta', function () {
 
   it('renders placeholder while loading', async function () {
     const wrapper = mountWithTheme(
-      <QuickTraceMeta
+      <WrappedQuickTraceMeta
         event={event}
         project={project}
         organization={organization}
```

Comment:
This `WrappedQuickTraceMeta` doesn't need `organization` as a parameter anymore due to the `useOrganization` hook right?